src/V3EmitCSyms.cpp

```diff
@@ -506,6 +507,22 @@ void EmitCSyms::emitSymHdr() {
     puts("\n// METHODS\n");
     puts("const char* name() { return TOP.name(); }\n");
 
+    if (v3Global.hasEvents()) {
+        puts("void enqueueTriggeredEventForClearing(VlEvent& event) {\n");
+        puts("#ifdef VL_DEBUG\n");
+        puts("if(!event.isTriggered()) {\n");
+        puts("VL_FATAL_MT(__FILE__, __LINE__, __FILE__, \"event passed to "
+             "'enqueueTriggeredEventForClearing' was not triggered\");\n");
+        puts("}\n");
+        puts("#endif\n");
+        puts("__Vm_triggeredEvents.push_back(&event);\n");
+        puts("}\n");
+        puts("void clearTriggeredEvents() {\n");
+        puts("for (const auto eventp: __Vm_triggeredEvents) eventp->clearTriggered();\n");
```

Comment:
```suggestion
        puts("for (const auto& eventp : __Vm_triggeredEvents) eventp->clearTriggered();\n");
```

Reply:
These are pointers to the reference would just slow things down. Fixed the spacing though.